docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  jupyterlab-extensions:
+    build:
+      context: ./src/dev/server
+    user: ${UID}
+    working_dir: /w
+    volumes:
+      - .:/w:cached
+      - ~/.config:/.config:cached
+    command: jupyter lab --port=8007 --allow-root --ip=0.0.0.0 --no-browser --notebook-dir=/w/jupyterlab --NotebookApp.token=''
```

Comment:
Odd that you would need --allow-root here since you're changing the UID. Are you sure this is needed?